src/Microsoft.ML.FastTree/TreeTrainersCatalog.cs

```diff
@@ -414,7 +414,7 @@ public static class TreeExtensions
         }
 
         /// <summary>
-        /// Predict a target using a decision tree regression model trained with the <see cref="FastForestBinaryTrainer"/> and advanced options.
+        /// Create <see cref="FastForestBinaryTrainer"/> with <see cref="FastForestBinaryTrainer.Options"/>, which predicts a target using a decision tree regression model.
```

Comment:
><see cref="FastForestBinaryTrainer.Options"/> [](start = 62, length = 45)

just say "with advanced options" to be consistent with the template

Reply:
Done.

---
In reply to: [277064586](https://github.com/dotnet/machinelearning/pull/3399#discussion_r277064586) [](ancestors = 277064586)